packages/astro/src/assets/utils/metadata.ts

```diff
@@ -1,8 +1,8 @@
 import { createRequire } from 'module';
 import fs from 'node:fs/promises';
-import { fileURLToPath } from 'node:url';
+import { fileURLToPath, pathToFileURL } from 'node:url';
 import { ImageMetadata, InputFormat } from '../types.js';
-const require = createRequire(import.meta.url);
+const require = createRequire(getModuleURL(import.meta.url));
 const sizeOf = require('image-size');
```

Comment:
Would using a normal import fix the problem? If so, it might be worth investigating that. I only used `require` to workaround a weird issue I had going on during dev, but it might not be required. The image integration actually imported the same package without a require

Reply:
Yeah I noticed that. I assumed it was necessary for a reason. I agree, I'll try that and see what happens.

Reply:
Switched to `import`, thanks for the suggestion